src/inverseRobot.js

```diff
@@ -7,7 +7,16 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newRob = {};
```

Comment:
1. [VARIABLES] - use variables for the main values so that you'll be able to reuse them and give them descriptive names.